README.md

```diff
@@ -854,6 +788,59 @@ The repo's collection can be easily imported into your Postman installation by o
 Postman and selecting File > Import... and choosing the Postman collection from your
 cloned DataGateway API repository.
 
+This collection has not been updated for the search API endpoints, so can only be used
+to query DataGateway API.
+
+
+
+
+# API Versioning
```

Comment:
It might be useful if you mention the `poetry run semantic-release print-version` command in this section.

Reply:
Resolved in d5d358c1f43573be7f0f443d4a20a9bd54a7c634